tinkerers/VABSYO.md

```diff
@@ -0,0 +1,69 @@
+# Yo. VABSYO Here.
+
+
+## My Introduction :-
+
+1. **I am everywhere**
+2. *I know everything*
+3. >Don't know what that makes me. I am comparable to God, I guess
+4. `Atleast, I am the creator of everything in this file.`
+
+### NOW YOU SEE ME :-
+<img src = https://raw.githubusercontent.com/Trajectory-Bootcamp/Trajectory-GitHub-Task/main/Pictures/seb_vettel.jpeg>
```

Comment:
Also here you need to change the part of the URL `.../main/Pictures/seb_vettel.jpeg` to `.../main/images/seb_vettel.jpeg`